### PR TITLE
data: Prashanth-bokkala

### PR DESCRIPTION
**Pull request by Prashanth-bokkala**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: data/Prashanth-bokkala.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
+{
+  "name": "Prashanth Bokkala",
+  "displayStatusPublic": true,
```
Comment (sital002) [inline data/Prashanth-bokkala.json:3]:
```suggestion
```
displayStatusPublic isnot required

File: data/Prashanth-bokkala.json [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  "type": "public",
```
Comment (sital002) [inline data/Prashanth-bokkala.json:4]:
```suggestion
```
type property isnot required

File: data/Prashanth-bokkala.json [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  "bio": "I'm a passionate frontend developer on a journey to master React. Currently immersed in the world of web development, I enjoy crafting user-friendly interfaces that bring ideas to life. Eager to learn and contribute, I'm dedicated to staying on top of the latest technologies. Let's build something amazing together!",
+  "avatar": "https://github.com/Prashanth-bokkala.png",
```
Comment (sital002) [inline data/Prashanth-bokkala.json:6]:
```suggestion
```
avatar property isnot required

**Review by sital002 — CHANGES_REQUESTED** [review]
Please make the necessary changes to get your profile merged.

File: data/Prashanth-bokkala.json (hunk 1 above)
Comment (Prashanth-bokkala) [reply data/Prashanth-bokkala.json:3]: Yes , but it's doesn't problem for the code & they shown on the tutorial that displayStatusPublic .

**Comment by Prashanth-bokkala** [comment]
> Please make the necessary changes to get your profile merged.

yes , but they shown in the tutorial about this so i included that , and mainly i checked the code with Playground in that , its shown that its a valid code , i checked twice before made pull request , if you still need changes please let me know

**Comment by sital002** [comment]
> > Please make the necessary changes to get your profile merged.
> 
> yes , but they shown in the tutorial about this so i included that , and mainly i checked the code with Playground in that , its shown that its a valid code , i checked twice before made pull request , if you still need changes please let me know

Yes those fields we required but now they are not required. The JSON playground only checks for where the json is valid or not.
image 
I will attach the link for json quick guide.
https://www.biodrop.io/docs/quickstart-json

**Review by sital002 — APPROVED** [review]
Looks good 👍
Welcome to Eddiehub community ✨✨

**Pull request merged**